x-pack/plugins/osquery/cypress/support/e2e.ts

```diff
@@ -23,17 +23,20 @@
 // ***********************************************************
 
 // force ESM in this module
-import type { SecuritySolutionDescribeBlockFtrConfig } from '@kbn/security-solution-plugin/scripts/run_cypress/utils';
-
 export {};
 
-import 'cypress-react-selector';
+// This module has some interesting ways of importing, check out the options in:
+// https://github.com/cypress-io/cypress/blob/0871b03c5b21711cd23056454da8f23dcaca4950/npm/grep/README.md#support-file
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
I'm not sure if this is what the above comment tries to explain, but why are you not using `@ts-expect-error`?

Reply:
I didn't know if there was a preference towards either of the directives. I can switch to `@ts-expect-error` if that's preferable.

Reply:
`@ts-expect-error` will make TypeScript give an error if the line below it no longer produces a TypeScript error. Whereas with `@ts-ignore` you're not notified if the error goes away

Reply:
Normally it's also good practise to limit it to the specific error you expect, so other errors you do not want to ignore will actually make the TypeScript check fail

Reply:
f1763b00f231d84ef010673f5d867cef4790d288